woocommerce/woocommerce-gutenberg-products-block
# Mini Cart: Fix footer buttons hover style

Fixes #6085

This PR applies the custom color settings to the hover state of the footer buttons.




### Screenshots

<img width="1372" alt="image" src="https://user-images.githubusercontent.com/5423135/159625281-de93570a-a974-465e-91b6-4456daa4021c.png">



### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Edit the Mini Cart Content Blocks.
2. Change the text and background colors.
3. Save. Go to the front end.
4. Add a product to open the Mini Cart Drawer.
5. Hover on the Cart and Checkout buttons to see the hover style uses new color settings.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
